### ISSUE TITLE
Bug: Discrepancy between KVS ERROR and KVS WARNING at column value

### ISSUE THREAD

**Issue by un1versal** [kvirc/KVIrc#2041]
I found a small discrepancy between **error** and **warning** on the **column** numbers.

made a scrip fail on purpose. to get both error and warning at same time.

```
[00:41:26] [KVS] Warning: Unterminated string constant
[00:41:26] [KVS]   In script context "sys", line 171, near character 71
[00:41:26] [KVS] Code listing:
[00:41:26] [KVS]   169 ...
[00:41:26] [KVS]   170      echo "      "/%aliasname <hdd> %S $tr("Output HDD/SSD information.")
[00:41:26] [KVS]   171      echo "      "/%aliasname <sen> %S $tr("Output Sensors information.")"
[00:41:26] [KVS]   172      echo "      "/%aliasname <share> <nick> %S $tr("Outputs a link to share %scriptname with a specific KVIrc user.")

[00:41:26] [KVS] Compilation error: Unexpected end of line in string constant (missing " character or unescaped newline)
[00:41:26] [KVS]   In script context "sys", line 171, near character 72
[00:41:26] [KVS] Code listing:
[00:41:26] [KVS]   169 ...
[00:41:26] [KVS]   170      echo "      "/%aliasname <hdd> %S $tr("Output HDD/SSD information.")
[00:41:26] [KVS]   171      echo "      "/%aliasname <sen> %S $tr("Output Sensors information.")"
[00:41:26] [KVS]   172      echo "      "/%aliasname <share> <nick> %S $tr("Outputs a link to share %scriptname with a specific KVIrc user.")
```

I cant remember is the desired pointer **near** or the **actual** character. (aside from what warning suggests?
### Expected behavior

That **Warning** and **Error** points to same near character **##**
### Actual behavior

**Warning points to** 

[00:41:26] [KVS]   In script context "sys", **line 171**, near character **71**

**Error points to:**
[00:41:26] [KVS]   In script context "sys", **line 171**, near character **72**

> ##### **Note**

If the desired is near the character then both are off due to another bug.
### Steps to reproduce the issue
1. Make a script fail so it gives you an error + warning at same time like adding and extra quote mark like below screenshot shows
2. trigger script
3. look at error and warning
### System information

```
KVIrc 4.9.2 'Aria'

Runtime Info:
System name: Linux 3.13.0-91-generic
System version: #138-Ubuntu SMP Fri Jun 24 17:00:34 UTC 2016
Architecture: x86_64
Qt version: 5.5.1
Qt theme: fusion

Build Info:
Build date: 2016-07-06 02:21:19 UTC
Sources date: 20160706
Revision number: git-6779-g96e790b
System name: Linux-3.13.0-91-generic
CPU name: x86_64
Build command: /usr/bin/cmake
Build flags: 
   MANDIR=share/man
   CMAKE_INSTALL_PREFIX=/usr
   LIB_SUFFIX=/x86_64-linux-gnu
   Threads=POSIX
Compiler name: /usr/bin/c++
Compiler flags: N/A
Qt version: 5.5.1
Features: IRC, Debug, Crypt, SSL, IfAddr, IPC, OSS, Transparency, Webkit, Perl, Python, Enchant, Qt5, KVS
```

**Comment by un1versal** [comment]
~~fixed by https://github.com/kvirc/KVIrc/commit/c67b91eb53f88d79daaed98765634bd3758c209a~~

**Comment by un1versal** [comment]
No this wasnt the one.... this is still an issue. its related to #2042 which was fixed by #2040 I closed this one in error, this is still valid last time I checked.

ok bed time Im half asleep

**Comment by wodim** [comment]
This is not really a bug. The warning points to the quotation mark, the error points to the newline character. For example:

    echo hello " hello

Results in:

	[12:47:58] [KVS] Warning: Unterminated string constant
	[12:47:58] [KVS]   In script context "c", line 1, near character 12
	[12:47:58] [KVS] Compilation error: Unexpected end of script in string constant (missing " character?)
	[12:47:58] [KVS]   In script context "c", line 1, near character 19

The unterminated string constant begins at character 12, and the newline character is found at character 19.
